src/renderer/library/components/dialog/ApiappAddForm.tsx

```diff
@@ -120,13 +122,26 @@ const ApiappAddForm = () => {
                                 e.stopPropagation();
                                 e.preventDefault();
                                 const v = searchInputRef.current?.value;
-                                if (!v) return ;
+                                if (!v) return;
                                 apiAppSearchAction(v);
                                 // e.preventDefault();
                                 // e.stopPropagation();
                             }
                         }}
                     />
+                    {searchInputRef ?
+                        <button
+                            className={stylesGlobal.button_clear}
+                            type="reset"
+                            onClick={() => {
+                                searchInputRef.current.value = "";
```

Comment:
Unlikely to occur in the React lifecycle, but I would still recommend to satisfy the TypeScript compiler in a project with stricter null/undefined checks:
`searchInputRef.current?.value = ...`

Reply:
TypeScript does not allow this: "La partie gauche d'une expression d'assignation ne doit pas être un accès à une propriété facultative.ts(2779)"

Reply:
Sorry, yes of course, I meant some code guard like:

`searchInputRef.current && searchInputRef.current.value = "";`

or

`if (searchInputRef.current) searchInputRef.current.value = "";`